docs/content/querying/segmentmetadataquery.md

```diff
@@ -96,18 +96,22 @@ This is a list of properties that determines the amount of information returned
 
 By default, all analysis types will be used. If a property is not needed, omitting it from this list will result in a more efficient query.
 
-There are 3 types of column analyses:
+There are four types of column analyses:
 
 #### cardinality
 
-* Estimated floor of cardinality for each column. Only relevant for dimension columns.
+* `cardinality` in the result will return the estimated floor of cardinality for each column. Only relevant for
+dimension columns.
 
 #### size
 
-* Estimated byte size for the segment columns if they were stored in a flat format
-
-* Estimated total segment byte size in if it was stored in a flat format
+* `size` in the result will contain the estimated total segment byte size as if the data were stored in text format
```

Comment:
i have no idea what this means.

Well I know what it means, but anyone not super familiar with Druid reading this would not.